dev/deno.ts

```diff
@@ -95,6 +95,25 @@ const generateRunnableFiles = generateFilepathsMatch({
   match: /\.(mjs|jsx?|tsx?)$/i,
 });
 
+const generateLintRules: Fig.Generator = {
+  script: "deno lint --rules",
+  getQueryTerm: ",",
+  cache: { ttl: 1000 * 60 * 60 * 24 * 3 },
+  postProcess: (out) => {
+    const lines = out.split("\n");
+    const suggestions: Fig.Suggestion[] = [];
+    for (const line of lines) {
+      if (line.startsWith(" - ")) {
+        suggestions.push({
+          name: line.slice(3),
+          icon: "fig://icon?type=string",
+        });
+      }
+    }
+    return suggestions;
```

Comment:
I think we can improve the readability of this:
```suggestion
    return out
      .split("\n")
      .filter(line => line.startsWith(" - "))
      .map(line => ({
        name: line.slice(3),
        icon: "fig://icon?type=string",
      }));
```